src/service/local.proto

```diff
@@ -35,11 +35,24 @@ message height_res {
 message region_req {}
 message region_res { int32 region = 1; }
 
+message add_gateway_req {
+  string owner = 1;
```

Comment:
these should be bytes to represent the pubkey_bin, not the b58 version (which I think you're implying here)

Reply:
should they be the b58 pubkeys? i see now that's what we've been sending over the ebus wire

Reply:
that may be true for ebus, but in grpc we send the pubkey_bin around since it has a binary type